### PR TITLE
consensus demo

### PR DESCRIPTION
**Pull request by chaychen2005**
1、Fake components for consensus
2、Add transaction to txPool periodically
3、Complete the consensus module (including initialization and startup)

### CODE REVIEW

File: fisco-bcos/Fake.h [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    {
+        /// set sig list
+        Signature sig;
+        h256 block_hash;
+        Secret sec = KeyPair::create().secret();
+        m_sigList.clear();
+        for (size_t i = 0; i < size; i++)
+        {
+            block_hash = sha3(toString("block " + i));
+            sig = sign(sec, block_hash);
```
Comment (wheatli) [inline fisco-bcos/Fake.h]: We need to fake multiple signatures on the same block hash, but not on different block hashes

File: fisco-bcos/Fake.h [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+using namespace dev;
+using namespace dev::blockchain;
+using namespace dev::eth;
+using namespace dev::sync;
+using namespace dev::blockverifier;
+
+class FakeBlock
+{
+public:
+    /// for normal case test
```
Comment (wheatli) [inline fisco-bcos/Fake.h]: Also we need a construction to fake a block by a parent block, to make them linked

File: fisco-bcos/Fake.h (hunk 1 above)
Comment (chaychen2005) [reply fisco-bcos/Fake.h]: Don't need faked block actually. So remove this part.

File: fisco-bcos/Fake.h (hunk 2 above)
Comment (chaychen2005) [reply fisco-bcos/Fake.h]: Don't need faked block actually. So remove this part.

**Pull request merged**